packages/compass-import-export/src/stores/export-store.ts

```diff
@@ -4,62 +4,38 @@ import type { Action, AnyAction } from 'redux';
 import { createStore, applyMiddleware, combineReducers } from 'redux';
 import type { ThunkAction } from 'redux-thunk';
 import thunk from 'redux-thunk';
-import {
-  globalAppRegistryActivated,
-  globalAppRegistry,
-  dataService,
-} from '../modules/compass';
-import {
-  dataServiceConnected,
-  dataServiceDisconnected,
-} from '../modules/compass/data-service';
-import { exportReducer, openExport } from '../modules/export';
+import { cancelExport, exportReducer, openExport } from '../modules/export';
 
-export function configureStore() {
+export function configureStore(services: ExportPluginServices) {
   return createStore(
     combineReducers({
       export: exportReducer,
-      globalAppRegistry,
-      dataService,
     }),
-    applyMiddleware(thunk)
+    applyMiddleware(thunk.withExtraArgument(services))
   );
 }
 
 export type RootExportState = ReturnType<
   ReturnType<typeof configureStore>['getState']
 >;
 
+export type ExportPluginServices = {
```

Comment:
Something for later, but I'm starting to think we should probably have some helper types to get those types easier from the returned plugin object maybe? Similar to `React.ComponentProps` or something like that